### PR TITLE
dropped z stream from python version in notebook

### PR DESCRIPTION
**Pull request by dibryant**
## Description

fixes: #49 

I have added` “cut -d '.' -f-2)”` to the original code  `"RUN sed -i -e "s/Python.*/$(python --version | cut -d '.' -f-2)\",/" /opt/app-root/share/jupyter/kernels/python3/kernel.json"
`in order to resolve the problem of the .z stream showing within the python version in  JL. 
## How Has This Been Tested?

https://github.com/opendatahub-io/notebooks/issues/49#issuecomment-1631234455
Screenshot 2023-07-13 at 3 52 24 PM (2)
## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by atheo89** [comment]
Built & run the minimal notebook and works as expected 
/lgtm 

image

File: jupyter/pytorch/ubi9-python-3.9/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN echo "Installing softwares and packages" && \
     micropipenv install && \
     rm -f ./Pipfile.lock && \
     # Replace Notebook's launcher, "(ipykernel)" with Python's version 3.x.y \
-    sed -i -e "s/Python.*/$(python --version)\",/" /opt/app-root/share/jupyter/kernels/python3/kernel.json
+    sed -i -e "s/Python.*/$(python --version | cut -d '.' -f-2s)\",/" /opt/app-root/share/jupyter/kernels/python3/kernel.json
```
Comment (harshad16) [inline jupyter/pytorch/ubi9-python-3.9/Dockerfile:21]:
```suggestion
    sed -i -e "s/Python.*/$(python --version | cut -d '.' -f-2)\",/" /opt/app-root/share/jupyter/kernels/python3/kernel.json
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
thanks for the work.
One change is needed

**Review by harshad16 — APPROVED** [review]
Thanks for the fix.

/lgtm
/approve

:100:

**Pull request merged**